hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -1600,6 +1600,16 @@
     </description>
   </property>
 
+  <property>
+    <name>ozone.om.max.pending.req.count</name>
+    <value>10000</value>
+    <tag>OZONE, OM</tag>
+    <description>the max pending om req count. the aim here is to limit the memory
+      token by om double buffer and the task queue of thread pool, which may lead to
+      OOM of OM
```

Comment:
Can you please improve this description a bit?

Reply:
thanks @adoroszlai for the reivew, i have improve the description, please take a look

Reply:
Thank you.